src/pages/index.js

```diff
@@ -6,61 +7,21 @@ import ModalWithForm from "../components/ModalWithForm.js";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
```

Comment:
Please, delete `this.disableButton();` from line 71 in `FormValidator` because you need to disable the button only in block `then` after successful response, otherwise it will be impossible to click the button for the 2nd time if there is a server error